packages/flutter/lib/src/material/search.dart

```diff
@@ -8,14 +8,14 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
 
 import 'app_bar.dart';
-import 'colors.dart';
 import 'debug.dart';
 import 'input_border.dart';
 import 'input_decorator.dart';
 import 'material_localizations.dart';
 import 'scaffold.dart';
 import 'text_field.dart';
 import 'theme.dart';
+import 'colors.dart';
```

Comment:
Undo this change

Reply:
I removed it since it is not needed anymore